runtimes/peregrine/src/lib.rs

```diff
@@ -692,52 +692,52 @@ construct_runtime! {
 		UncheckedExtrinsic = UncheckedExtrinsic,
 	{
 		// Basic stuff; balances is uncallable initially.
-		System: frame_system::{Pallet, Call, Config, Storage, Event<T>} = 0,
-		RandomnessCollectiveFlip: pallet_randomness_collective_flip::{Pallet, Storage} = 1,
+		System: frame_system = 0,
+		RandomnessCollectiveFlip: pallet_randomness_collective_flip = 1,
 
-		Timestamp: pallet_timestamp::{Pallet, Call, Storage, Inherent} = 2,
+		Timestamp: pallet_timestamp = 2,
 		Indices: pallet_indices::{Pallet, Call, Storage, Event<T>} = 5,
-		Balances: pallet_balances::{Pallet, Call, Storage, Config<T>, Event<T>} = 6,
+		Balances: pallet_balances = 6,
 		TransactionPayment: pallet_transaction_payment::{Pallet, Storage} = 7,
-		Sudo: pallet_sudo::{Pallet, Call, Config<T>, Storage, Event<T>} = 8,
+		Sudo: pallet_sudo = 8,
 
 		// Consensus support.
 		// The following order MUST NOT be changed: Authorship -> Staking -> Session -> Aura -> AuraExt
 		Authorship: pallet_authorship::{Pallet, Call, Storage} = 20,
-		ParachainStaking: parachain_staking::{Pallet, Call, Storage, Event<T>, Config<T>} = 21,
-		Session: pallet_session::{Pallet, Call, Storage, Event, Config<T>} = 22,
+		ParachainStaking: parachain_staking = 21,
+		Session: pallet_session = 22,
 		Aura: pallet_aura::{Pallet, Config<T>} = 23,
 		AuraExt: cumulus_pallet_aura_ext::{Pallet, Config} = 24,
```

Comment:
M: Add Storage here or use the _new way_. I validated this by checking in the [Statemine configuration](https://github.com/paritytech/cumulus/blob/master/polkadot-parachains/statemine/src/lib.rs#L731).

Reply:
8521b59